modules/javafx.graphics/src/main/docs/javafx/scene/doc-files/cssref.html

```diff
@@ -680,6 +682,80 @@ <h3><a id="introinheritance">Inheritance</a></h3>
         inherited provided there is not an author or an inline-style that applies specifically to that node.
         In this case, the inherited font is created from the user-set font and any parts of the applicable
         author or in-line style.
+
+    <h3><a id="introtransitions">Transitions</a></h3>
+    <p>JavaFX supports <em>implicit transitions</em> for properties that are styled by CSS. When a property value is
+        changed, it smoothly transitions to the new value over a period of time. Implicit transitions are supported
```

Comment:
Maybe not so smoothly when using a step interpolator?

Reply:
I've changed the wording a bit.